target_decisioning_engine/utils.py

```diff
@@ -46,33 +38,41 @@ def get_rule_key(rule):
     return rule.get("ruleKey")
 
 
-def _get_default_parsed_url(url):
-    """Returns default dict for urls that cannot be parsed"""
-    parsed = dict(DEFAULT_PARSED_URL)
-    parsed["url"] = url
-    return parsed
-
-
 def parse_url(url):
     """parse url"""
     if not is_string(url):
-        return _get_default_parsed_url(EMPTY_STRING)
+        return {
+            "url": "",
```

Comment:
Instead of an empty string, I think we can still at least return this dict with "url": url